tap_recharge/client.py

```diff
@@ -131,14 +146,18 @@ def check_access_token(self):
         response = self.__session.get(
             # Simple endpoint that returns 1 record w/ default organization URN
             url='https://api.rechargeapps.com',
-            headers=headers)
+            headers=headers,
+            timeout=self.request_timeout)
         if response.status_code != 200:
             LOGGER.error('Error status_code = %s', response.status_code)
             raise_for_error(response)
         else:
             return True
 
-
+    @backoff.on_exception(
```

Comment:
Add it like below
@backoff.on_exception(
        backoff.expo,
        Timeout,
        max_tries=5,
        factor=2)

Reply:
> Add it like below
> 
> ```
> @backoff.on_exception(
>         backoff.expo,
>         Timeout,
>         max_tries=5,
>         factor=2)
> ```

Added factor in the backoff